apps/demos/Demos/Pagination/Overview/React/App.tsx

```diff
@@ -1,13 +1,36 @@
-import React from "react";
-import Pagination from "devextreme-react/pagination";
-import { customers } from "./data.ts";
-
-const columns = ["CompanyName", "City", "State", "Phone", "Fax"];
+import React, { useState } from 'react';
+import Pagination from 'devextreme-react/pagination';
+import EmployeeGallery from './EmployeesGallery.tsx';
+import { employees } from './data.ts';
 
 const App = () => {
+  const [pageSize, setPageSize] = useState(4);
+  const [pageIndex, setPageIndex] = useState(1);
+
   return (
     <>
-      <Pagination />
+      <EmployeeGallery
+        employees={employees}
+        pageSize={pageSize}
+        pageIndex={pageIndex}
+      />
+      <Pagination
+        showInfo
+        showNavigationButtons
+        allowedPageSizes={[4, 6]}
```

Comment:
Let's move array out of component to make it stable (so that it will always have same ref). Our components have problems with updating, so moving all complex objects from component is best practice for us

Reply:
fixed